apps/docs/src/basics/accessibility.md

```diff
@@ -0,0 +1,77 @@
+
+![image](/assets/structure.png)
+
+Recognizing that each added element increases UI complexity, strive for simplicity by incorporating clearly visible elements, adequate contrast and size, a hierarchical layout, and key information readily discernable at a glance. Organize important actions strategically, placing them at the screen's top or bottom for accessibility, and group related items with similar importance side by side. This approach ensures a more intuitive and user-friendly UI interpretation.
+
+## Color & contrast
+
+In adhering to AA/AAA accessibility standards, the color palette is meticulously preconfigured and readily available for implementation. However, achieving high contrast relies not just on color selection but on utilizing the appropriate shades within the chosen colors. It's imperative to strike a balance by employing two complementary colors — one from the lighter spectrum and another from the darker spectrum of the palette — especially when dealing with text and background elements.
+
+![image](/assets/contrast.png)
+
+Furthermore, an effective strategy involves adopting the traffic light color system to harmonize content and visuals seamlessly. This system, inspired by the familiar traffic light colors, ensures a coherent and visually intuitive combination, enhancing not only accessibility but also the overall user experience.
+
+## Feedback
+
+Ensuring a transparent and communicative user experience involves providing timely feedback on screen interactions. It's essential for users to be informed about the success or failure of their actions. Employing diverse feedback mechanisms amplifies user understanding, and incorporating components such as toast messages, notifications, error labels, and icons proves highly effective.
+
+The use of a toast component offers subtle, non-intrusive messages that appear briefly, notifying users without disrupting their flow. Notifications, on the other hand, provide more detailed and persistent feedback, offering clarity on the outcome of their interactions.
+Error messaging through labels plays a crucial role in guiding users when issues arise, presenting information in a clear and comprehensible manner. Icons serve as visual indicators, enhancing the communicative aspect of feedback.
+
+![image](/assets/feedback.png)
+
+In addition, integrating these feedback components with the traffic light color system adds a layer of accessibility refinement. Aligning feedback with this system—using colors like green for success, yellow for warnings, and red for errors—creates a cohesive and universally recognizable visual language. This not only ensures users are promptly informed about their actions but also contributes to an inclusive and user-friendly interface.
```

Comment:
Thats a good point. Unfortunatelly I cannot access this file anymore because it is no longer in my new branch. Maybe we can have a look at this afterwards.

Reply:
You need to switch to the `docs/jannick` branch and change it there